cli/src/commands/operation/log.rs

```diff
@@ -130,7 +130,8 @@ fn do_op_log(
                 &text,
                 OperationTemplateLanguage::wrap_operation,
             )?
-            .labeled("op_log");
+            .labeled("log")
+            .labeled("operation");
```

Comment:
nit: should be `.labeled("operation").labeled("op_log")` (= `["op_log", "operation"]`)? (read: the command is "op_log", and the context object is an "operation".)

If we apply the same rule to `jj log`, the labels will be `["log", "commit"]`.

Reply:
Are we going to remove other `<command_name>` labels (e.g. "log", "bookmark_list")? If not, I would avoid derailing the existing convention here.

Reply:
It sounds like you're OK with the new state of this, so I'll merge it.

I agree that it'd good to keep in mind the entire convention and change the whole thing at once, if somebody feels up to it (which I don't at the moment). So, thank you for keeping track of what the current convention is and how often it's used.

I'd also change the convention, ideally, but I'm not going to do it in the short-term. Also, I think it'd be more important to make the labels more easily discoverable, though you did point out some downsides of putting them all in `templates.toml` and I'm not sure what other options are viable. Neither of these is an urgent task.

Reply:
Actually, I'll wait a bit, since this is not urgent, and since it's possible that you actually meant for me to change something (though I don't think so). I'll merge this in a few hours or tomorrow if I don't hear otherwise.

Reply:
Wrong order? should be "op_log" -> "operation". Other than that, it looks good to me, thanks.

Regarding discoverability, maybe we should document the current convention in templates.md, and suggest using `--color=debug`.